Plugins~/Src/MeshSyncClientBlender/msblenBinder.cpp

```diff
@@ -74,6 +91,8 @@ extern PropertyRNA* BlenderPyDepsgraphObjectInstance_object;
 
 extern PropertyRNA* BlenderPyDepsgraph_object_instances;
 
+extern PropertyRNA* BlenderPyContext_depsgraph_object_instances;
```

Comment:
This should be removed, it has been replaced by BlenderPyDepsgraph_object_instances

Reply:
Fixed